content/300-guides/300-upgrade-guides/800-upgrade-from-prisma-1/08-upgrade-from-mongodb-beta.mdx

```diff
@@ -1,16 +1,14 @@
 ---
 title: 'Upgrade from MongoDB Beta'
-metaTitle: 'Upgrade from the Prisma 1 MongoDB Beta to Prisma 2'
-metaDescription: 'Learn how to upgrade your MongoDB application running Prisma 1 to Prisma 2.'
+metaTitle: 'Upgrade from the Prisma 1 MongoDB Beta to Prisma 4'
+metaDescription: 'Learn how to upgrade your MongoDB application running Prisma 1 to Prisma 4.'
```

Comment:
This also applies to people upgrading to Prisma 2 and 3 still. We sometimes call this `2+` or `2 and later` or similar. @tanberry might know our conventions here.

Reply:
Fair point! I'll amend it